include/mbedtls/rsa.h

```diff
@@ -1,11 +1,12 @@
 /**
  * \file rsa.h
  *
- * \brief The RSA public-key cryptosystem.
+ * \brief This file contains RSA definitions and functions.
```

Comment:
Rephrased as "This file defines the RSA public-key cryptosystem." is this accurate?

Reply:
I don't like this wording very much because it sounds as if the document was defining RSA. What about "This file provides an API for the RSA public-key cryptosystem."?

Reply:
I like this. Thanks for updating the description @RosZad !